kouta-backend/src/main/scala/fi/oph/kouta/domain/toteutus.scala

```diff
@@ -236,7 +236,7 @@ case class Toteutus(oid: Option[ToteutusOid] = None,
                     teemakuva: Option[String] = None,
                     modified: Option[Modified] = None,
                     koulutusMetadata: Option[KoulutusMetadata] = None,
-                    koulutuksetKoodiUri: Seq[String] = Seq(),
+                    koulutuksetKoodiUri: Option[Seq[String]] = None,
```

Comment:
Onko tää Option-kääräisymuutos tarpeellinen? Eikö tyhjä sekvenssi sisällä käytännössä saman tiedon kuin None?

Reply:
Liittyy siis tähän: https://github.com/Opetushallitus/kouta-external/pull/65 ks. kommentti koodissa

Reply:
Tuli mieleen, et onko pienempi paha vaan antaa kentän propagoitua externaliin, jos ei oikeasti tarvita eroa keissien "arvoja ei ole" ja "en tiedä onko arvoja" välillä. Mut eihän tää maailman vakavin asia oo, turhaa monimutkaisuutta vaan.

Reply:
Oon kyllä ihan samaa mieltä, että monimutkaistaa tämän kouta-backendin toteutusta. Kouta-external on kuitenkin "toissijainen" sovellus, jonka pitäisi muuttua kouta-backendin ehdoilla eikä toisinpäin. Otetaanpa @pretseli mukaan tähän pohdintaan: onko muutos tarpeellinen? Viittaan siis tähän: https://github.com/Opetushallitus/kouta-external/pull/65

Reply:
Onhan tämä kieltämättä vähän huono ajatus näin kokeneemmilla aivoilla nähtynä 😅 . Jos tehdään silleen, että koetan muotoilla tiketin noista kouta-externalin testien parantamisesta, koska siellähän tämä varmaan pitäisi korjata. Eli nämä muutokset voinee jättää tässä vaiheessa pois.

Reply:
Nyt ois revertattu kyseinen muutos sekä täältä että kouta-externalista.